vrpn_Magellan.C

```diff
@@ -62,6 +62,8 @@ vrpn_Magellan::vrpn_Magellan (const char * name, vrpn_Connection * c,
 	vrpn_Button::num_buttons = _numbuttons;
 	vrpn_Analog::num_channel = _numchannels;
 
+        vrpn_gettimeofday(&timestamp, NULL);	// Set watchdog now
```

Comment:
Extra space

Reply:
My editor automatically replaces tabs with spaces.  They display the same for tab stops of 8 but will vary for different-sized tab stops.